### PR TITLE
Task11 - notes functionality (add, show)

### PR DESCRIPTION
**Pull request by PavloRohozhyn**
added notes
 - new command note-add
 - new command note-show
 - update the help page
note - use a new file for storage data (notes.pkl)

### CODE REVIEW

**Comment by PavloRohozhyn** [comment]
FYI: @sazhukov , @valeriia1421

File: src/contactsbot/notesOperations.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+from .Notes import Notes
+from .helpers import print_with_color
+
+
+def note_add_handler(note: Notes):
+    """ note add hendler """
+    print_with_color('Please enter a note text : ', 'yellow')
+    note_text = input()
+    note.note_add(NoteRecord(note_text))
+    return "note added"
```
Comment (MaksVeter) [inline src/contactsbot/notesOperations.py:12]: Please capitalize it

**Review by MaksVeter — CHANGES_REQUESTED** [review]
The note-show command display extra `[]`
image

File: src/contactsbot/notesOperations.py (hunk 1 above)
Comment (PavloRohozhyn) [reply src/contactsbot/notesOperations.py:12]: what are the capitalize, could you please provide example 
ASAP

Comment (PavloRohozhyn) [reply src/contactsbot/notesOperations.py:12]: and [ ] - it is a feature! will be integrate in next commit (but now I will remove it)

Comment (MaksVeter) [reply src/contactsbot/notesOperations.py:12]: capitalize - is making first letter uppercase

Comment (PavloRohozhyn) [reply src/contactsbot/notesOperations.py]: fixed

**Review by MaksVeter — DISMISSED** [review]

**Comment by PavloRohozhyn** [comment]
@MaksVeter  my `merge commit was removed
`

**Pull request merged**